elements/pfe-badge/BaseBadge.scss

```diff
@@ -2,4 +2,6 @@
   position: relative;
   white-space: nowrap;
   text-align: center;
+  box-sizing: border-box;
```

Comment:
Looks like you should be able to just use display: inline-block here, the box-sizing at least from what I can see doesn't seem to change the width.